### PR TITLE
feature: reorganize analytics site

### PR DESCRIPTION
**Pull request by drew-harris**

### CODE REVIEW

**Comment by drew-harris** [comment]
image

**Comment by drew-harris** [comment]
image

**Comment by densumesh** [comment]
image
when theres a generate off chunks this seems to overflow

image
make this table wider

Should make a new issue to make the dedicated query page to view a single query and its results

**Comment by densumesh** [comment]
image

the bottom timestamps should correspond to the time range i.e. show the date rather than time stamp if looking at past week

Also if there is only 1 or 2 points make the graph have artifiically larger padding in terms of time so that the points aren't just at the side and are in the middle

**Comment by densumesh** [comment]
image

Head queries should have a top bar showing what each row is as well

**Comment by drew-harris** [comment]
Working now and bundle size is fixed. LGTM!

File: server/src/data/models.rs [hunk 1]
```diff
@@ -3736,9 +3736,7 @@ pub enum SearchAnalytics {
         sort_order: Option<SortOrder>,
     },
     #[schema(title = "CountQueries")]
-    CountQueries {
-        filter: Option<SearchAnalyticsFilter>,
```
Comment (densumesh) [inline server/src/data/models.rs:3740]: This should remain the same no?

**Review by densumesh — CHANGES_REQUESTED** [review]

**Review by densumesh — APPROVED** [review]

**Review by densumesh — APPROVED** [review]

**Pull request merged**